File: files-ui/test_commands_args/TestCommandArgs.py
#! python3

# special vars should be available when project is open
# and this code belongs to the project
# and NOT available otherwise
print(f"{__rhino_command__}\n")
print(f"{__rhino_doc__}\n")
print(f"{__rhino_runmode__}\n")
print(f"{__is_interactive__}\n")

File: files-ui/test_commands_args/TestCommandArgs.cs
// #! csharp
using System;

// special vars should be available when project is open
// and this code belongs to the project
// and NOT available otherwise
Console.WriteLine(__rhino_command__ ?? "None");
Console.WriteLine(__rhino_doc__);
Console.WriteLine(__rhino_runmode__);
Console.WriteLine(__is_interactive__);